manim/utils/tex_file_writing.py

```diff
@@ -116,7 +116,7 @@ def generate_tex_file(
 
 def tex_compilation_command(
```

Comment:
Since this function no longer returns a `str` with the TeX command, but instead returns a `list[str]` containing the command arguments, could we change this function's name to something like `prepare_tex_compilation_args`?

Reply:
In the synopsys of [subprocess.run](https://docs.python.org/3/library/subprocess.html#subprocess.run), the name of the first and only positional argument, `args`, is a bit misleading because its first item is the command name. I think it follows the C language tradition where the first item of the argument vector (argv) of [exec](https://www.man7.org/linux/man-pages/man3/exec.3.html) and `main(int argc, char *argv[])` is the command name too.
If I saw a function named `prepare_tex_compilation_command_args` I would assume it returns the arguments only, not the command. I suggest we simply name this function `make_tex_compilation_command`.

Reply:
`make_tex_compilation_command` sounds fine to me :+1: